File: cml-compiler/cml-generator/src/test/resources/templates/lang/java/association/organization.java
class Organization
{
    private static Employment employment;

    private final String name;

    public Organization(String name, List<Employee> employees)
    {
        this.name = name;

        employment.linkMany(this, employees);
    }

    public String getName()
    {
        return this.name;
    }

    public List<Employee> getEmployees()
    {
        return employment.employeesOf(this);
    }

    public String toString()
    {
        return new StringBuilder(Organization.class.getSimpleName())
                   .append('(')
                   .append("name=").append(String.format("\"%s\"", getName()))
                   .append(')')
                   .toString();
    }

    static void setEmployment(Employment association)
    {
        employment = association;
    }

    static
    {
        Employment.init(Organization.class);
    }
}

File: cml-compiler/cml-generator/src/test/resources/templates/lang/py/association/organization.py
class Organization:

    _employment = _Employment()

    def __init__(self, name: 'str', employees: 'List[Employee]') -> 'None':
        self.__name = name

        self._employment.link_many(self, employees)

    @property
    def name(self) -> 'str':
        return self.__name

    @property
    def employees(self) -> 'List[Employee]':
        return self._employment.employees_of(self)

    def __str__(self) -> 'str':
        return "%s(name=%s)" % (
            type(self).__name__,
            self.name
        )